src/DefaultUnleash.php

```diff
@@ -205,4 +224,35 @@ private function findStrategyHandlers(Strategy $strategy): array
 
         return $handlers;
     }
+
+    private function isDependencyEnabled(FeatureDependency $dependency, Context $context, bool $default): bool
```

Comment:
can we adjust domain vocabulary to `isParentDependencySatisfied`? The reason for that is that we want to emphasize this is a parent dependency not child dependency. Also dependency doesn't have to be enabled - the spec also defines disabled parent as an option.

Reply:
Done.